app/services/articles/feed.rb

```diff
@@ -126,7 +133,7 @@ def score_experience_level(article)
 
     def globally_hot_articles(user_signed_in)
       hot_stories = published_articles_by_tag.
-        where("score > ? OR featured = ?", 9, true).
+        where("score > ? OR featured = ?", 8, true).
```

Comment:
This is unrelated but I think we can drop this a bit given that we have a bit better spam scoping in place.